farmdata2_modules/fd2_tabs/fd2_field_kit/seedingInput.html

```diff
@@ -29,15 +29,15 @@ <h1 style='text-align:center;'>Seeding Input Log</h1>
                 <dropdown-with-all data-cy="area-selection" :dropdown-list='areaFilter' @selection-changed='setNewArea' style='width: 125px;'>Area:<label style='color: #da4f3f'>*</label></dropdown-with-all>
                 <br>
                 <label>Rows/Bed:<label style='color: #da4f3f'>*</label> <input data-cy='row-bed' style='height: 25px; width: 4em;' type='number' v-model='numRowBed'></label>
-                <label><dropdown-with-all data-cy='unit-feet' :default-input='selectedFeetUnit' :dropdown-list='feetUnits' @selection-changed='setNewFeetUnit' style='padding-left:20px;' ></dropdown-with-all>
+                <label><dropdown-with-all data-cy='unit-feet' :selected-val='selectedFeetUnit' :dropdown-list='feetUnits' @selection-changed='setNewFeetUnit' style='padding-left:20px;' ></dropdown-with-all>
                 <label style='color: #da4f3f'>*</label> <input data-cy='num-feet' style='height: 25px; width: 4em;' type='number' v-model='numFeet'></label>
             </div>
        </fieldset>   
        <fieldset class="panel panel-default center-block" style="max-width: 500px;">
             <legend class="panel-heading" style='background-color: #da4f3f; color: white;font-size: x-large;'>Labor</legend>
             <div class="center-block" style='padding: 12px;text-align:center;font-size: medium; margin-top: 45px;'>
                 <label style='color: #da4f3f'>*&nbsp;</label><input data-cy="num-workers" style='height: 25px; width: 4em;' type='number' v-model='numWorkers'><label>&nbsp; Worker for </label> 
-                <label><label style='color: #da4f3f'>*</label> <input data-cy='time-spent' style='height: 25px; width: 4em;' type='number' v-model='time'> <dropdown-with-all data-cy='time-unit' style='text-align:center;' :default-input='selectedTimeUnit' :dropdown-list='timeUnits' @selection-changed='setNewTimeUnit'></dropdown-with-all></label>
+                <label><label style='color: #da4f3f'>*</label> <input data-cy='time-spent' style='height: 25px; width: 4em;' type='number' v-model='time'> <dropdown-with-all data-cy='time-unit' style='text-align:center;' :selection-val='selectedTimeUnit' :dropdown-list='timeUnits' @selection-changed='setNewTimeUnit'></dropdown-with-all></label>
```

Comment:
Should this be selected-val?

Reply:
Might want to use a search feature in your IDE to be sure you find all instances of default-input and also selection-val to replace.

Reply:
I just did a search for both _default-input_ and _selection-val_ on al the files I changed to make sure there were no more of them. We should be good to go now.